modules/statistics/statistics.js

```diff
@@ -104,23 +106,32 @@ function Statistics(xmpp, options) {
             && (Statistics.disableThirdPartyRequests !== true);
     if (this.callStatsIntegrationEnabled) {
         loadCallStatsAPI(this.options.callStatsCustomScriptUrl);
+
+        if (!this.options.callStatsConfIDNamespace) {
```

Comment:
You probably added this because I told you in the previous review. I have since noticed another problem: we don't have a proper window.location in React Native, but actually, it makes sense to use `this.options.config.hosts.domain` in `JitsiConference`, so it's always set and to the right value.